docs/source/usage/cli.rst

```diff
@@ -243,9 +243,18 @@ serial console information for a node from the Bare Metal service as follows:
 | console_info    | {u'url': u'``tcp://<host>:<port>``', u'type': u'socat'}              |
 +-----------------+----------------------------------------------------------------------+
 
+If ``console_enabled`` is ``true``, we can access the serial console using following command:
+
+``socat - tcp:<host>:<port>``
+
 If ``console_enabled`` is ``false`` or ``console_info`` is ``None`` then
 the serial console is disabled. Note, there can only be one ipmi connection to the node, meaning only one user may access the console at a time.
 
+Note that for serial console to function:
+
+* The ``ipmi_terminal_port`` must be set by the admin. This port must be unique per node/controller.
```

Comment:
After further experimentation, it's safest to just say that the port must be unique.

Reply:
Ah, okay will make changes!